src/keyboard.cpp

```diff
@@ -1398,3 +1401,25 @@ void InputMapper::CPCkeyFromJoystickAxis(SDL_JoyAxisEvent jaxis, dword *cpc_key,
 }
 
 InputMapper::InputMapper(t_CPC *CPC): CPC(CPC) { }
+
+void applyKeypress(dword cpc_key, byte keyboard_matrix[], bool pressed) {
+    if ((!CPC.paused) && (static_cast<byte>(cpc_key) != 0xff)) {
+		if (pressed) {
```

Comment:
Fixed. BTW, it would be great to reindent the whole thing: different indentation rules in different part of the code (e.g. main indentation different from gui indentation). This is easy to do via indent or similar tool, but would create a gigantic commit...